src/components/Asset/AssetDetails.tsx

```diff
@@ -70,3 +77,46 @@ export const AssetBalance: FC<AssetBalanceProps> = ({
     </Skeleton>
   )
 }
+
+export const AssetBalanceWithDollarValue: FC<AssetBalanceWithDollarProps> = ({
```

Comment:
I think that it would be better if you created instead just an `AssetBalanceUsd` component. 

That way, you can compose `AssetBalance` and `AssetBalanceUsd` components as needed when building UI. Having the styling tightly coupled here is less flexible.

Reply:
done